tests/CMakeLists.txt

```diff
@@ -57,8 +58,11 @@ if(QUDA_ARPACK)
 endif()
 
 # define tests
+add_executable(c_interface_test c_interface_test.c)
+  target_link_libraries(c_interface_test ${TEST_LIBS})
+  quda_checkbuildtest(c_interface_test QUDA_BUILD_ALL_TESTS)
 
-# if we build with QDP JIT the tests cannot run anyway
+  # if we build with QDP JIT the tests cannot run anyway
```

Comment:
Space inserted before `#`

Reply:
Is there a particular reason that's a problem in the context of `CMakeLists.txt`, since it's a comment (and thus should arguably match indentation), not a `pragma` in C++?

Reply:
We indent comments elsewhere in the file, so I'm going to opt to leave it as-is.

Reply:
My point was this is an unnecessary change made in this PR.  We should be striving to minimizing unnecessary diffs.

Reply:
Ah, fair. It looks like it got picked up because it was neighboring an addition Mathias made. I'll revert it.

Reply:
Actually, it looks like the issue is more that the two lines above got indented for no good reason.

Reply:
Fixed